### DISCUSSION TITLE
Why do we check for .rej files?

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [MartinBernstorff/swift-python-cookiecutter#75] [General]
It is probably worth documenting this workflow if we plan to keep it. The error is also somewhat unhelpful.

[check_for_rej.yml](https://github.com/MartinBernstorff/swift-python-cookiecutter/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/.github/workflows/check_for_rej.yml)

**Comment by MartinBernstorff** [comment]
`.rej` files occur when `cruft update` could not merge two files. They need to be handled, but are easy to miss if there's no CI catching them :-)
 
Should we just copy the above explanation into the workflow?

**Reply by KennethEnevoldsen** [reply]
Ahh that is perfect. I added it and merged it in
